Marlin/example_configurations/wt150/Configuration_adv.h

```diff
@@ -551,6 +551,13 @@
   // To have any effect, endstops must be enabled during SD printing.
   //#define ABORT_ON_ENDSTOP_HIT_FEATURE_ENABLED
 
+  // This option allows you to easily reprint the same SD Card file as
+  // was last printed.  At the end of a print, the LCD Menu will jump
+  // straight to the file previously selected.  A single click of the encoder
+  // wheel will restart the print.  Another file or LCD option can be
+  // selected by using the encoder wheel to navigate through the menu structure.
```

Comment:
For longer comments Marlin has moved to favoring blocks.

In terms of the interface design, it may be friendlier to have a menu with "Print again" and "Done" options rather than simply opening the SD card menu to the last selection.

Reply:
This scheme is pretty 'friendly'.  I would think an extra menu item just adds clutter and the whole reason for this is to support people that are using their 3D Printers to print parts.   They can break the part off the glass and press the button and it does the next one with no fuss.

> For longer comments Marlin has moved to favoring blocks.

I was trying to preserve the style of the previous SD Card options up above it.